src/api/app/views/webui/user/user_profile_redesign/_basic_info.html.haml

```diff
@@ -18,3 +28,15 @@
       = mail_to(user.email, title: "Send mail to #{user.name}", class: 'd-block') do
         %i.fas.fa-envelope.mr-1
         = user.email
+
+
+:javascript
+  $('#toggle-in-place-editing').on('click', function () {
+    $('.in-place-editing .basic-info').toggleClass('d-none');
+    $('.in-place-editing .editing-form').toggleClass('d-none');
+  });
+  $('#cancel-in-place-editing').on('click', function () {
+    $('.in-place-editing .basic-info').toggleClass('d-none');
+    $('.in-place-editing .editing-form').toggleClass('d-none');
+    return false;
```

Comment:
```suggestion
```

Reply:
This is needed to prevent the wrapping form to be sent and update the changes.

Reply:
Oh right, because `Cancel` is a button.